build.xml

```diff
@@ -398,6 +425,13 @@
         <mkdir dir="${build.dir.lib}"/>
         <mkdir dir="${jacoco.export.dir}"/>
         <mkdir dir="${jacoco.partials.dir}"/>
+
+        <!-- Set up jdk specific properties -->
+        <javac includes="**/JdkProperties.java" srcdir="test/anttasks" destdir="${test.classes}" includeantruntime="true" source="${java.default}" target="${java.default}">
```

Comment:
why do we need explicit includes? don't we want to build whatever is in `test/anttasks` ?

Reply:
Good question, it probably doesn't matter too much. Let's keep it aligned with Apache for now. Less conflicts

Reply:
This looks like a bug to me, can you please replace includes to cover all the files? I cannot really see the reason to not compile all files from there while I can see reasons to compile everything :)

Reply:
It is explicitly adding whatever classes we have/need there.

The explicit compilation for this particular class was added here because we cannot trigger the build without it. Otherwise, all tasks in that anttasks package are compiled at a later point. You can see after building that they are all there. So I do not see a bug.
We cannot run  `ant jar` without this particular test. At the same time some of those classes in that folder are also dependent on other classes not in anttasks. For instance `TestNameCheckTask` and `EchoEclipseProjectLibs`.

Btw it seems in CASSANDRA-18618 it was you adding explicitly `SetSystemProperty` to this target and not just all classes :D